talk/owt/sdk/p2p/p2ppeerconnectionchannel.cc

```diff
@@ -525,17 +539,6 @@ void P2PPeerConnectionChannel::OnMessageTracksAdded(
         if (it_publishing != publishing_streams_.end())
           publishing_streams_.erase(it_publishing);
       }
-      // Trigger the successful callback of publish.
```

Comment:
why did the base framwork move away from this? It makes sense to only fire the on_success for Publish once the front-end sends the "chat-tracks-added"

Reply:
I actually agree that it makes the most sense to delay the success until chat-tracks-added is received. I don't know why the base framework went away from this approach, but the publish_success_callbacks_ vector wasn't event being used in the code, which is why I just removed it.